en/stacks/tpm/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: TPM
+---
+
+# Trusted Platform Module (TPM)
+
+There are currently two different snaps that help manage TPM under Ubuntu Core Stacks
```

Comment:
I need some help on this. I agree with you, I just have no experience in the details. Could also reach out to Scott to write a little blurb up.